lich.rbw

```diff
@@ -1544,16 +1565,35 @@ class XMLParser
 
   DECADE = 10 * 31_536_000
 
+def init_psm3_dialog(kind)
```

Comment:
this is a unnecessary, breaking change that needs to be avoided.

`Id` and `Name` can co-exist in a flat hash, if you want to maintain some sort of mapping, we should add a new hash called `id_name` or something.

Reply:
I'm still exploring the simplification discussion, and may opt to follow this path however (see Lich Dev discord) I have demonstrated this is not 'breaking'.  I would appreciate your review and concurrence then mark the discussion resolved purely from the 'breaking' context.

Reply:
With simplification suggested, this init procedure is not required.  Agreed, removing, resolved.